core/ip/rest_resolver.go

```diff
@@ -52,13 +52,17 @@ func NewResolverWithTimeout(ipifyUrl string, timeout time.Duration) Resolver {
 	}
 }
 
+type ipResponse struct {
```

Comment:
imho bad name here, should be resultingIP or returnedIP

Reply:
Not my scope. And all structures in tequilapi endpoints has `*Response` and `*Request` naming